internal/local/structure_test.go

```diff
@@ -336,6 +336,73 @@ exports = function({ query }) {
         }
     }
 }
+`, string(rule))
+	})
+
+	t.Run("should write services with v2 Rules to disk", func(t *testing.T) {
+		mdbSvcName := "mdbSvc"
+		data := []ServiceStructure{{
+			Config: map[string]interface{}{
+				"name":    mdbSvcName,
+				"type":    "mongodb",
+				"config":  map[string]interface{}{},
+				"version": 1,
+			},
+			DefaultRule: map[string]interface{}{
+				"roles": []interface{}{
+					map[string]interface{}{
+						"name": "owner",
+						"apply_when": map[string]interface{}{
+							"userId": "%%user.id",
+						},
+						"read": true,
+					},
+				},
+			},
+			Rules: []map[string]interface{}{
+				{
+					"name":       "foo.bar",
```

Comment:
would bump the need to remove this (and other instances of it) so we more accurately reflect how the server exports these payloads